docs/model-dev-guide/debug-models.rst

```diff
@@ -36,8 +36,7 @@ Determined model. Follow the nine steps as applicable to your environment:
  Prerequisite
 **************
 
-Successful cluster installation as described in :ref:`Install the Determined cluster
-<install-cluster>`.
```

Comment:
I like that it's shorter, but I kinda like the cross link...

Just a thought, feel free to ignore.

Reply:
Agree. The cross link went to Step 6 Install the Determined Cluster in the "old" basic.rst file. And that step basically said nothing except the following:

*****************************************
 Step 6 - Install the Determined Cluster
*****************************************

A Determined cluster comprises a master and one or more agents. The cluster can be installed on
Amazon Web Services (AWS), Google Cloud Platform (GCP), on-premise, or on a local development
machine.

So, I'll keep some of that content and also direct the user to the cluster deployment guide.